### DISCUSSION TITLE
Docs: use NavLink instead of UnstyledButton

### DISCUSSION THREAD

**Discussion by ShahriarKh** [mantinedev/mantine#2124] [Feedback]
In docs, AppShell examples, instead of using `UnstyledButton`,  isn't it better to use the `NavLink` which is a similar component that already exists?
https://github.com/mantinedev/mantine/blob/master/src/mantine-demos/src/demos/core/AppShell/_mainLinks.tsx

Pull request: #2125

**Comment by slk333** [comment]
I agree that this is a bit confusing! Most people will look at the code and wonder why the Navlink was not used instead
